### ISSUE TITLE
[feature]: Specify custom port other than 80

### ISSUE THREAD

**Issue by markskram** [makeplane/plane#1140]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

I would like to run Plane on a port other than 80. Port 80 is already in use by another server app. It looks like others would like the same thing, as commented here: https://github.com/makeplane/plane/issues/718

### Why should this be worked on?

A server often runs multiple services at once that each have their own web interfaces. They need to run on different ports so they can all be accessed.

**Comment by pablohashescobar** [comment]
Hello, @markskram! Unfortunately, we don't have a direct configuration option to change the default port 80 at the moment. However, you can modify the host port on nginx in the docker-compose.yml and docker-compose-hub.yml files to achieve this.

**Comment by require-dev** [comment]
Wouldn't changing the `80:80` portmapping in the docker-compose.yml & docker-compose-hub.yml to `${NGINX_PORT}:80` fix this issue in the current setup? 

This way you can add NGINX_PORT=80 to the `.env.example` which would allow easy changing of the port, without actually changing the original docker-compose or docker-compose-hub.yml?

**Comment by pablohashescobar** [comment]
Thank you for pointing that out! We will make sure to include this in the .env.example.

**Comment by pablohashescobar** [comment]
Created a PR for the above configuration #1144

**Comment by markskram** [comment]
I cherry-picked the change and verified it works. I am running plane on port 8111 now.

I am experiencing a separate issue where the database connection fails on `django.db.utils.OperationalError: connection to server at "plane-db" (172.19.0.5), port 5432 failed: FATAL:  password authentication failed for user "plane"`. This might be unrelated.

**Comment by pablohashescobar** [comment]
Hello @markskram, I wanted to inform you that we have made some changes to our Docker setup. In order to proceed, please provide the database username, password, and host. You can find the default values in [.env.example](https://github.com/makeplane/plane/blob/develop/.env.example) file. We appreciate your cooperation!

**Comment by markskram** [comment]
What would be an appropriate value for `PGHOST`? I've tried all sorts of values 'localhost', '196.168.2.147' (the devices local IP address), `host.docker.internal`, or even leaving it as `plane-db`. They all give me "Connection refused" when planebackend attempts to connect on port 5432.

**Comment by markskram** [comment]
I am attempting to use the postgres that comes with the docker setup. Should I be running my own?

**Comment by markskram** [comment]
So if I leave it as `plane-db`, which makes it use the container ip, it is able to connect. but it still is not able to authenticate with the default password specified in .env.example. What should I be setting that to?

I am using the new settings in `.env.example`
```
# Database Settings
PGUSER="plane"
PGPASSWORD="plane"
PGHOST="plane-db"
PGDATABASE="plane"
```

`django.db.utils.OperationalError: connection to server at "plane-db" (172.25.0.4), port 5432 failed: FATAL:  password authentication failed for user "plane"`

**Comment by markskram** [comment]
I got the database working by deleting the Volumes in Docker and running the installation steps again. Now I'm farther along. 

The signin page does not work with the default credentials. I'm having a similar issue as https://github.com/makeplane/plane/issues/1107.

Login is successful if I run a browser on the same machine as docker. If I try to connect from another computer on the network, login fails.

I have tried the following:
`./setup.sh`
`./setup.sh localhost`
`./setup.sh 192.168.1.123` (ip address for machine running Docker)
`./setup.sh machine_name.local` (FDQN of machine running Docker)

Is there a different address I should specify? Maybe one for one of the containers?

**Comment by pablohashescobar** [comment]
Hi @markskram, you will have to provide the public IP address (./setup.sh http://<INSTANCE_IP> if you are running this in ` develop```` branch else only ./setup.sh <INSTANCE_IP> if you are in `master` branch) of the instance you are running docker.

**Comment by pablohashescobar** [comment]
Closing this issue as the PR is already merged #1144
